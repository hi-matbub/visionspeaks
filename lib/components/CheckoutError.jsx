import React from 'react';
import PropTypes from 'prop-types';

const CheckoutError = ({ children }) => <>{children}</>;

CheckoutError.propTypes = {
  children: PropTypes.node.isRequired,
};

export default CheckoutError;
