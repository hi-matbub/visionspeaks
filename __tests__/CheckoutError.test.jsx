import React from 'react';
import ReactDOM from 'react-dom';
import renderer from 'react-test-renderer';
import CheckoutError from '../lib/components/CheckoutError';

it('CheckoutError component renders smoke test without crashing', () => {
  const div = document.createElement('div');
  ReactDOM.render(<CheckoutError>Hello</CheckoutError>, div);
  ReactDOM.unmountComponentAtNode(div);
});

it('CheckoutError component renders unchanged', () => {
  const tree = renderer.create(<CheckoutError>Hello</CheckoutError>).toJSON();
  expect(tree).toMatchSnapshot();
});
