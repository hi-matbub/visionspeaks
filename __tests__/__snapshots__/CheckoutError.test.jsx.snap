// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CheckoutError component renders unchanged 1`] = `"Hello"`;
